titus_isolate/isolate/utils.py

```diff
@@ -11,3 +14,15 @@ def get_static_workloads(workloads):
 
 def get_workloads_by_type(workloads, workload_type):
     return [w for w in workloads if w.get_type() == workload_type]
+
+
+def get_allocator_class(config_manager):
+    alloc_str = config_manager.get(ALLOCATOR_KEY)
+
+    if alloc_str not in ALLOCATOR_VALUES:
+        alloc_str = DEFAULT_ALLOCATOR
```

Comment:
should we only do that if `alloc_str` is `None`, but `throw` otherwise?

Reply:
It's a good point.  On receiving an unexpected allocator, what should we do?  First question is crash or not?  Usually I say crash, but since this is fed by a fast property that can cause very wide spread changes I'm hesitant.  I should at least emit an error message here.

I'll do that.  I'll add an error message and keep the behavior that we fallback to the default allocator on unexpected input.